[2024-09-12T02:15:24.578+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T02:15:24.620+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V6.third_task scheduled__2021-09-11T00:00:00+00:00 [queued]>
[2024-09-12T02:15:24.655+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V6.third_task scheduled__2021-09-11T00:00:00+00:00 [queued]>
[2024-09-12T02:15:24.663+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-12T02:15:24.720+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): third_task> on 2021-09-11 00:00:00+00:00
[2024-09-12T02:15:24.740+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=911) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-12T02:15:24.744+0000] {standard_task_runner.py:72} INFO - Started process 923 to run task
[2024-09-12T02:15:24.753+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V6', 'third_task', 'scheduled__2021-09-11T00:00:00+00:00', '--job-id', '18445', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpful34n44']
[2024-09-12T02:15:24.770+0000] {standard_task_runner.py:105} INFO - Job 18445: Subtask third_task
[2024-09-12T02:15:24.996+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V6.third_task scheduled__2021-09-11T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-12T02:15:25.503+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='acap' AIRFLOW_CTX_DAG_ID='our_first_dag_V6' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2021-09-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-09-11T00:00:00+00:00'
[2024-09-12T02:15:25.507+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T02:15:25.553+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-12T02:15:25.556+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the third task!']
[2024-09-12T02:15:25.577+0000] {subprocess.py:86} INFO - Output:
[2024-09-12T02:15:25.579+0000] {subprocess.py:93} INFO - hey, this is the third task!
[2024-09-12T02:15:25.580+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-12T02:15:25.633+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T02:15:25.638+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V6, task_id=third_task, run_id=scheduled__2021-09-11T00:00:00+00:00, execution_date=20210911T000000, start_date=20240912T021524, end_date=20240912T021525
[2024-09-12T02:15:25.775+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T02:15:25.820+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
